Tech Lead/Minutes/20230112.md

```diff
@@ -0,0 +1,50 @@
+## Technical Lead Call 12 January 2023
+
+### Date and participants
+
+- Names legend:
+  - Bouwe Andela [@bouweandela](https://github.com/bouweandela)
+  - Klaus Zimmermann [@zklaus](https://github.com/zklaus)
+  - Remi Kazeroni [@remi-kazeroni](https://github.com/remi-kazeroni)
+  - Saskia Sloosveldt [@sloosvel](https://github.com/sloosvel)
+  - Valeriu Predoi [@valeriupredoi](https://github.com/valeriupredoi)
+  - Manuel Schlund [@schlunma](https://github.com/schlunma)
+  - Breixo Solino [@bsolino](https://github.com/bsolino)
+- Date: 12 January 2023 2pm UK time (3pm CEST)
+- Participants: @bouweandela @zklaus @remi-kazeroni @sloosvel @valeriupredoi @schlunma @bsolino
+- Chairs and minutes takers: @valeriupredoi and @remi-kazeroni
+- Agenda in [comment](https://github.com/ESMValGroup/Community/discussions/68#discussioncomment-4667572)
+
+### Topics discussed
+
+Topics added to the [discussion item page](https://github.com/ESMValGroup/Community/discussions/68)
+
+1. From last call, still to be discussed/addressed:
+
+  - common Gmail address to share credentials for services used to develop ESMValTool. See [here](https://github.com/ESMValGroup/Community/discussions/62#discussioncomment-4119360) (@valeriupredoi: fuuuu... forgot again, will do TODAY ie 12 January 2023)
+  - *Documentation* @valeriupredoi will update the release documentation early January, previous release managers can then take a look
+  - [Release procedure: documentation of the release process](https://github.com/ESMValGroup/Community/discussions/62#discussioncomment-4390307)
+  - [Release procedure: Strategy to run all recipes](https://github.com/ESMValGroup/Community/discussions/62#discussioncomment-4119346)
+  - [Release procedure: Administration of the VM and its usage during the release](https://github.com/ESMValGroup/Community/discussions/62#discussioncomment-4148563)
+
+2. [Opening Slack for the general public](https://github.com/ESMValGroup/Community/discussions/68#discussioncomment-4399940):
+  - @bouweandela, @zklaus: would be positive about that
+  - @valeriupredoi, @remi-kazeroni: have some reservations that users would stop using github to report issues.
+  - @saskia: experience from other projects not very positive
+  - Opening access: to be discussed with the other teams. As a first step we could open access to other teams and structure our Slack a bit more.
+
+3. [Discussing 2.8 release](https://github.com/ESMValGroup/Community/discussions/68#discussioncomment-4668496):
+   - @bouweandela
+     - a bit slower since review is at a premium - ESMValGroup/ESMvalCore#1609 is close to be merged (dataset versioning is important);
```

Comment:
```suggestion
     - a bit slower since review is at a premium - ESMValGroup/ESMvalCore#1877 is close to be merged (dataset versioning is important);
```